moped-editor/src/views/projects/projectView/ProjectSummary/ProjectSummaryKnackDataTrackerSync.js

```diff
@@ -142,21 +156,20 @@ const ProjectSummaryKnackDataTrackerSync = ({
     // this code is ready to "re-sync" a project thanks to its use of a dynamic knackHttpMethod
     console.log("HTTP method: " + knackHttpMethod);
     // POST (create) or PUT (update) a project record in Knack
-    const signalIds = signals.map(signal => signal.knack_id);
     return fetch(knackProjectEndpointUrl, {
```

Comment:
Unrelated to this PR, but one of the three records that I sync'd didn't update in the UI until I refreshed the page. Could this leave open the possibility of someone creating duplicate Knack project records? Not familiar enough with this feature but wanted to call it out in case we need to make a backlog issue.

Reply:
interesting — i also thought i might have noticed the `refetch()` callback not reliably triggering a page refresh :/

seems like a race condition in our fetch chaining, but i can't spot an obvious problem

Reply:
and in fact there are many dupe projects in the Data Tracker 😞 
- https://github.com/cityofaustin/atd-data-tech/issues/11224